File: delete.js
const fs = require('fs');
const path = require('path');

const targetPath = path.resolve(__dirname, 'target');
if (fs.existsSync(targetPath)) {
    console.log("Deleting deeply nested target directory using Node...");
    fs.rmSync(targetPath, { recursive: true, force: true });
    console.log("Deleted successfully.");
} else {
    console.log("Not found.");
}

File: delete.py
import shutil
import os

path = r"\\?\C:\Users\sif-\Desktop\All projects\SugarCane Supplier\target"
if os.path.exists(path):
    print("Deleting long path...")
    shutil.rmtree(path)
    print("Deleted.")
else:
    print("Path not found.")
